### PR TITLE
Corrections for chapter 10 of episode 10

### PR DESCRIPTION
**Pull request by wikimatze**

### CODE REVIEW

File: 10/chapter-06.srt [hunk 1]
```diff
@@ -224,6 +224,9 @@ So I will print current stanza here.
 00:04:19,330 --> 00:04:25,290
 And maybe the reps as well so we can check it.
 
+
+4:25 - Drew: Sure.
```
Comment (wikimatze) [inline 10/chapter-06.srt:228]: Don't know in which way new sentences are injected, but here I think we need this.

File: 10/chapter-06.srt [hunk 2]
```diff
@@ -242,7 +245,7 @@ SP: No, it's not.
 
 55
 00:04:41,980 --> 00:04:47,200
-In theory it should stop.
+So in therory, it should stop.
```
Comment (nelstrom) [inline 10/chapter-06.srt:248]: "In theory" looks good to me!

**Comment by nelstrom** [comment]
Great job @wikimatze, thanks!

File: 10/chapter-06.srt (hunk 1 above)
Comment (nelstrom) [inline 10/chapter-06.srt:228]: I just merged this with the previous caption: https://github.com/fromPeerToPeer/captions/pull/75/commits/23eb81a1a0eb39bc2aac2af1ad36e931ea4b38f8

File: 10/chapter-06.srt (hunk 2 above)
Comment (wikimatze) [inline 10/chapter-06.srt:248]: Okay, you decide in the end :).

File: 10/chapter-06.srt (hunk 1 above)
Comment (wikimatze) [inline 10/chapter-06.srt:228]: Fine!

**Comment by wikimatze** [comment]
It's always a pleasure to help you. Keep on going with the video production.

**Pull request merged**